server/src/test/java/org/elasticsearch/action/IndicesRequestIT.java

```diff
@@ -140,10 +140,7 @@ protected int minimumNumberOfReplicas() {
     @Override
     protected Settings nodeSettings(int ordinal) {
         // must set this independently of the plugin so it overrides MockTransportService
-        return Settings.builder().put(super.nodeSettings(ordinal))
-            // InternalClusterInfoService sends IndicesStatsRequest periodically which messes with this test
-            // this setting disables it...
-            .put("cluster.routing.allocation.disk.threshold_enabled", false).build();
+        return Settings.builder().put(super.nodeSettings(ordinal)).build();
```

Comment:
I'm very much not sure about this. I couldn't get the `IndicesStatsRequest` to mess with this test as the comment indicated, but that doesn't mean it's not possible, just that I couldn't work it out. @s1monw can you help?